includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -1096,6 +1145,24 @@ public static function print_status_meta_box( $post ) {
 					</div>
 
 					<div class="misc-pub-section">
+						<?php
+						$staleness = self::get_post_staleness( $post );
+						if ( ! empty( $staleness ) ) {
+							echo '<div class="notice notice-info notice-alt inline"><p>';
+							echo '<b>';
+							esc_html_e( 'Stale results', 'amp' );
+							echo '</b>';
+							echo '<br>';
+							if ( ! empty( $staleness['theme'] ) && ! empty( $staleness['plugins'] ) ) {
+								esc_html_e( 'Different theme and plugins were active when these results were obtained. Please re-check.', 'amp' );
+							} elseif ( ! empty( $staleness['theme'] ) ) {
+								esc_html_e( 'A different theme was active when these results were obtained. Please re-check.', 'amp' );
+							} elseif ( ! empty( $staleness['plugin'] ) ) {
```

Comment:
@westonruter Shouldn't this key be `'plugins'`?  And this conditional has the same concern of `isset()` as above.

Reply:
Good catch. Fixed in 6b5cb8be.